src/spikeinterface/core/tests/test_recording_tools.py

```diff
@@ -32,6 +32,9 @@ def test_get_noise_levels():
     noise_levels = get_noise_levels(rec, return_scaled=True)
     print(noise_levels)
 
+    noise_levels = get_noise_levels(rec, return_scaled=True, method="std")
```

Comment:
Thanks for adding a tests to this. I don't see the point of printing here though (I know the other one is doing it).

I think this test could use the NumpyExtractor using a trace whose statistics we know (because we created) and test that the noise level is the one that was used to generate the traces. 

What do you think?

Reply:
Yeah I didn't understand printing either, but just kept it for consistency.

Makes sense, we could also check mad that way by simply generating a `NumpyRecording` that follows a gaussian of known standard deviation :)